arm-batch/2015-12-01/swagger/BatchManagement.json

```diff
@@ -1481,7 +1465,7 @@
         "details": {
           "type": "array",
           "items": {
-            "$ref": "#/definitions/ErrorDetail"
+            "$ref": "#/definitions/CloudError"
```

Comment:
ErrorDetail did not contain the "details" array. Whereas the modified one says that it can have an optional "details" property of type array. 

Since the property is optional, it is fine. However wanted to bring it to your notice.